test/framework/src/main/java/org/elasticsearch/repositories/blobstore/AbstractBlobContainerRetriesTestCase.java

```diff
@@ -341,9 +348,12 @@ public void testReadBlobWithPrematureConnectionClose() {
         });
         assertThat(
             exception.getMessage().toLowerCase(Locale.ROOT),
-            either(containsString("premature end of chunk coded message body: closing chunk expected")).or(
-                containsString("premature end of content-length delimited message body")
-            ).or(containsString("connection closed prematurely"))
+            anyOf(
+                containsString("premature end of chunk coded message body: closing chunk expected"),
+                containsString("premature end of content-length delimited message body"),
+                containsString("connection closed prematurely"),
+                alwaysFlushBody ? never() : containsString("the target server failed to respond")
```

Comment:
Could you add a comment here?

It looks like if we flush the output, we expect the original errors. Without flushing the output stream, then we always get a standard error message that the node is gone?

Reply:
Does e6d486bd1c2173006477413d799f92870c4c6e7b help?

Reply:
Yep, thanks